.github/workflows/repo-automator.yml

```diff
@@ -19,6 +19,7 @@ on:
 jobs:
   Validate:
     runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' && ! github.event.pull_request.head.repo.fork }}
```

Comment:
This will not run on pushes. Please help me.

Reply:
On push there are no labels 😲

Reply:
Updated condition to run this on push 👍